src/components/Label.js

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Text from './Text';
+
+/*
+ * renders text on top of children components
+ */
+class Label extends PureComponent {
+  static get propTypes() {
+    return {
+      text: PropTypes.string,
+      children: PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.arrayOf(PropTypes.node),
+      ]),
+      textClasses: PropTypes.string,
+      stacked: PropTypes.bool,
+      className: PropTypes.string,
+      theme: PropTypes.object,
+      style: PropTypes.object,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      className: '',
+      textClasses: '',
+      stacked: true,
+      theme: {},
+      style: {},
+    };
+  }
+
+  render() {
+    const {
+      text,
+      children,
+      className,
+      theme,
+      style,
+      textClasses,
```

Comment:
I think everywhere else in the codebase we use singular nouns to describe things, ie `textClass`

Reply:
I'm not seeing any use cases of `textClass` in the repo actually. There is one instance of `classNames` plural though in the Table (rowClassNames to be exact). how about `textClassNames`?

Reply:
Maybe i've been spending too much time in `bwallet` haha. That is fine with me